app/layout.tsx

```diff
@@ -22,8 +26,69 @@ export default function RootLayout({
       <body
         className={`${jetBrainsMono.className} antialiased bg-base text-text`}
       >
+        <header className="max-w-[1280px] w-full mx-auto px-4 pt-8 flex items-center justify-between pb-8">
```

Comment:
Try to use the `styled()` util from `react-tailwind-variants` for long classes like this

```tsx
const Header = styled("header", {
  base: "max-w-[1280px] w-full mx-auto px-4 pt-8 flex items-center justify-between pb-8"
});
```

You can also do

```tsx
<Flex className="max-w-[1280px] !pl-0" p={8} center width="full" asChild>
  <header>...</header>
</Flex>
```